packages/angular/src/card/card-title.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, HostBinding, Input } from '@angular/core';
+
+@Component({
+  selector: 'ai-card-title',
+  template: `
+    <div class="iot--card--title--text" [attr.title]="text">
```

Comment:
Should we put an ngIf to allow them to completely replace this with custom content from ng-content?

Reply:
I figure that if they need something that custom they can build the structure themselves, eventually this will have some extra features (like carbon tooltips for truncated text) and trying to merge that with a fully custom content will become a bit of a pain.

The `ai-card-header` (and I mean, really all of these components) is built with the expectation that consumers could pass in whatever they want ... I fully expect consumers to build their own header components from these building blocks.

So yeah, I suppose we could do that, but we might not have to?

Reply:
Ah, true. Maybe if it was a more complicated component.